nbtc/relayer.go

```diff
@@ -43,17 +51,21 @@ func NewRelayer(btcClientConfig rpcclient.ConnConfig, processTxsInterval time.Du
 		processTxsInterval = time.Second * 5
 	}
 
+	if confirmTxsInterval == 0 {
+		confirmTxsInterval = time.Second * 7
```

Comment:
Can you explain what this is?

Reply:
So basically if the value passed is 0, we set it by deafult to 7 seconds. This is how often the relayer will query the node to update the broadcasted transactions status